themes/socialbase/templates/comment/field--comment.html.twig

```diff
@@ -30,7 +30,8 @@
 {# extra condition to make sure the card is not shown when there are no comments and comments are closed #}
 {% if comments or comment_form %}
 
-  <section{{ attributes.setAttribute('id', 'section-comments') }}>
+  <a id="section-comments"></a>
+  <section{{ attributes }}>
```

Comment:
This could negatively affect styling of our own code or extending projects. It's also not needed if this is used to focus the user's viewport. `id` attributes on any element can be used for this :)

Reply:
We need this to allow this section to have it's own id based on the drupal render array. That's how Ajax comments works and knows what section to replace after a comment is being made. So the entire comment field is updates including the counts etc.

Adding